hoot-core/src/main/cpp/hoot/core/cmd/StatCmd.cpp

```diff
@@ -31,13 +31,13 @@
 #include <hoot/core/io/OsmMapReaderFactory.h>
 #include <hoot/core/elements/OsmMap.h>
 #include <hoot/core/visitors/SingleStatistic.h>
-#include <hoot/core/elements/ConstElementVisitor.h>
+#include <hoot/core/elements/ElementVisitor.h>
 #include <hoot/core/util/Exception.h>
 #include <hoot/core/io/OsmMapReader.h>
 #include <hoot/core/util/ConfigOptions.h>
 #include <hoot/core/util/Configurable.h>
 #include <hoot/core/io/PartialOsmMapReader.h>
-#include <hoot/core/io/ConstElementVisitorInputStream.h>
+#include <hoot/core/io/ElementVisitorInputStream.h>
 
```

Comment:
Was able to get rid of the PITA to maintain second set of streaming classes for ConstElementVisitor.